gtsam/linear/NoiseModel.h

```diff
@@ -18,6 +18,7 @@
 
 #pragma once
 
+#include<iostream>
```

Comment:
Nonono! Iosfwd is *reluctantly* tolerated, iostream is not. We want as few headers included in .h files as possible, to cut down on compile time. Especially a header like noiseModel.

Also, header order in GTSAM is near to far, so iosfwd (IF needed) would come last.

Reply:
Woops. I was using that for debugging and accidentally left it in. :disappointed: